app/views/users/login_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<main>
+  <% if @user.errors.any? %>
+      <ul class="errors">
+        <% @user.errors.each do |column, message| %>
+          <li>
+            <strong><%= column.capitalize %></strong> <%= message %>
+          </li>
+        <% end %>
+      </ul>
+    <% end %>
```

Comment:
Displaying the validation errors should be done with flash notices in the future.